channel-sdk/src/main/java/com/zello/channel/sdk/SessionContext.kt

```diff
@@ -14,10 +18,19 @@ import com.zello.channel.sdk.platform.Encoder
  */
 internal interface SessionContext {
 
+	val transportFactory: TransportFactory
+	val locationManager: LocationManager
```

Comment:
Seems to miss a line break here for consistency.

Reply:
Done